src/cdn_definitions/data.yaml

```diff
@@ -194,3 +194,7 @@ exclude_from_listings:
 # future. Versions not listed here are exempt from various tooling.
 rhel_open_dist:
 - 0
+
+# If set to False cdn-utils will set ubi_population repo note to False for all DOT repos.
```

Comment:
Minor: 'cdn-utils' is an unknown reference here as it's an internal project, doesn't have any related information or reference in this repo and might add to confusion here. It could just be "If set to False, ubi_population repo note will be False and no UBI DOT repo will be populated."

Reply:
Thanks for the comment. Done.